tests/helpers/testers.py

```diff
@@ -33,7 +33,7 @@
     pass
 
 NUM_PROCESSES = 2
-NUM_BATCHES = 10
+NUM_BATCHES = 8  # Need to be divisible with the number of processes
```

Comment:
shall we co even with a smaller number?
```suggestion
NUM_BATCHES = 4  # Need to be divisible with the number of processes
```

Reply:
lets try